include/exadg/utilities/print_functions.h

```diff
@@ -82,7 +85,10 @@ template<typename ParameterType>
 void
 print_value(dealii::ConditionalOStream const & pcout, ParameterType const value)
 {
-  pcout << value << std::endl;
+  if constexpr(std::is_enum_v<ParameterType> || magic_enum::is_scoped_enum_v<ParameterType>)
+    pcout << Utilities::enum_to_string(value) << std::endl;
```

Comment:
Is there a possibility to also hide `std::is_enum_v` and `magic_enum` here (with the goal that all the `magic_enum` occurrences are wrapped in ExaDG? What is the meaning of this line of code (it is hard to read for me right now)? Comments would help here.

Reply:
Sure, I can add e.g. a function ```Utilities::is_enum<Type>()``` and add a comment in its definition

Reply:
sound great

Reply:
let us check once more why this discussion has been resolved

Reply:
It seems I introduced mentioned function but did not apply it here. Sorry for that. I fixed this.